src/components/reusable/table/table-toolbar.scss

```diff
@@ -0,0 +1,19 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  padding: var(--kd-spacing-8) var(--kd-spacing-0);
+}
+.title {
+  @include typography.type-body-01;
+  font-weight: var(--kd-font-weight-bold);
+  color: var(--kd-color-text-primary);
+  margin-right: var(--kd-spacing-16);
+}
+
+.slot-wrapper {
+  height: 52px;
```

Comment:
Any particular reason for this fixed height?

Reply:
It's 52px for the content in the design, and again we can update this later when the design for it gets revamped.